class-job.php

```diff
@@ -51,6 +51,8 @@ public function save() {
 			$data['interval'] = $this->interval;
 		}
 
+		wp_cache_delete( "jobs:{$this->site}", 'cavalcade-jobs' );
```

Comment:
But aren't we avoiding this issue by using a non-persistent cache group? We only cache the cron schedule once per request and it's actually never stored in a persistent object cache?